src/phel/core.phel

```diff
@@ -677,6 +690,32 @@ arrays. Use (php/aunset ds key)" )))
     (php/aunset x key)
     x))
 
+(defn union
+  "Union multiple sets"
+  [& sets]
+  (apply concat sets))
```

Comment:
This method fails, if we just call `(union)`.

Reply:
I believe this also means that `union` should always create a new set instead of updating the first one? And that this rule should apply to `intersection` and `difference` too?